Libraries/Opc.Ua.Server/Session/SessionManager.cs

```diff
@@ -423,7 +413,6 @@ public virtual void CloseSession(NodeId sessionId)
                     m_server.ServerDiagnostics.CurrentSessionCount--;
```

Comment:
This does not seem to get called anymore

Reply:
good catch, it checks for session.id but the dict key is not the session Id.

Reply:
fixed